### DISCUSSION TITLE
Voicepacks installation on Viomi SE

### DISCUSSION THREAD

**Discussion by rufik** [Hypfer/Valetudo#1703] [Q&A / Support]
Viomi SE on release 2023.01.0. I have generated voicepack using this generator, got `pl.pkg` (polish lang) package. Then I hosted file on https://www.file.io, put link and `pl` lang code (hash left empty), pressed "Set voice pack" and...nothing happend.
Does this feature work on Viomi SE? Or is there a way to install voice manually?

![image](https://user-images.githubusercontent.com/425558/216958505-e1459f46-bdac-4b2a-bd21-f31f8523f342.png)

**Comment by Aeris1One** [comment]
Hi,
I manually replaced "/usr/share/audio/english" wav files, the robot still think it speaks english while it speaks french to me.
```bash
cd /mnt/UDISK/Download
ls # make sure you don't have any .wav file in here, also you can clean up some _tmp files that may be left when trying to install voicepacks through the ui
wget https://builder.dontvacuum.me/pkg/voice/viomi/v6/[languagecode].zip
unzip [languagecode].zip 
mv *.wav /usr/share/audio/english
rm *.wav
```
That isn't persistent on factory reset

**Reply by rufik** [reply]
Thanks a lot! I did that and it works fine 🙂
But quickly got into trouble because the overlay partition was 100% used. Viomi SE has small disk space unfortunately.
